crates/nu-utils/src/sample_config/default_config.nu

```diff
@@ -205,6 +205,7 @@ $env.config = {
         quick: true    # set this to false to prevent auto-selecting completions when only one remains
         partial: true    # set this to false to prevent partial filling of the prompt
         algorithm: "prefix"    # prefix or fuzzy
+        sort: "default" # "default" (choose based on algorithm) or "alphabetical"
```

Comment:
When a user sees this i'm not sure they'll know the difference between default and alphabetical. I'm wondering if there's something better than "default" that can spell out what it does better.

Reply:
How about "smart"? (sorry, been away for a while)

Reply:
ya, smart seems better. good choice.